[DOCS] Blocklist (#1811)
elastic/security-docs: docs/getting-started/configure-integration-policy.asciidoc

* <<adv-policy-settings>>
* <<save-policy>>

4. Click the **Trusted applications**, **Event filters**, **Host isolation exceptions**, and **Blocklist** tabs to review the endpoint policy artifacts assigned to this integration policy (for more information, refer to <<trusted-apps-ov>>, <<event-filters>>, <<host-isolation-exceptions>>, and <<blocklist>>). On these tabs, you can:
* Expand and view an artifact — Click the arrow next to its name.

> * View an artifact's details — Click the actions button (**...**), then select **View full details**.

Author: I'll do a find/replace in my local repo then push the changes to GitHub.

+
TIP: Platinum and Enterprise customers can customize these notifications using the `Elastic Security {action} {filename}` syntax.

> Malware protection also allows you to implement a blocklist to prevent specified applications from running on hosts, extending the list of processes that {endpoint-sec} considers malicious. Use the **Blocklist enabled** toggle to enable or disable this feature for all hosts associated with the integration policy. To configure the blocklist, refer to <<blocklist>>.

Reviewer: > Malware protection also allows you to implement a blocklist...

Maybe a better word here is "manage" in place of "implement" ?

suggestion:
> Malware protection also allows you to manage a blocklist...

Author: ```suggestion
Malware protection also allows you to manage a blocklist to prevent specified applications from running on hosts, extending the list of processes that {endpoint-sec} considers malicious. Use the **Blocklist enabled** toggle to enable or disable this feature for all hosts associated with the integration policy. To configure the blocklist, refer to <<blocklist>>.
```

> Malware protection also allows you to enable a blocklist of specified applications that are prevented from running in host environments, extending the list of processes that Endpoint Security considers malicious. Use the **Blocklist enabled** toggle to enable or disable this feature for all hosts associated with the integration policy. To configure specific applications on the blocklist, refer to <<blocklist>>.

Reviewer 2: ```suggestion
Malware protection also allows you to enable a blocklist of specified applications that are prevented from running in host environments, extending the list of processes that Endpoint Security considers malicious. Use the **Blocklist enabled** toggle to enable or disable this feature for all hosts associated with the integration policy. To manage applications on the blocklist, refer to <<blocklist>>.
```

Author: Going with a more concise version of the final sentence (in comment above), as the reference doesn't need to be that specific.

> Malware protection also allows you to enable a blocklist of specified applications that are prevented from running in host environments, extending the list of processes that Endpoint Security considers malicious. Use the **Blocklist enabled** toggle to enable or disable this feature for all hosts associated with the integration policy. To configure specific applications on the blocklist, refer to <<blocklist>>.

Author: Combining elements from both reviews:
```suggestion
Malware protection also allows you to implement a blocklist to prevent specified applications from running on hosts, extending the list of processes that {endpoint-sec} considers malicious. Use the **Blocklist enabled** toggle to enable or disable this feature for all hosts associated with the integration policy. To configure the blocklist, refer to <<blocklist>>.
```
